src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Storage/BlobContainers/AzureBlobContainer.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Kaiyuanshe.OpenHackathon.Server.Storage.BlobContainers
+{
+
+    public interface IAzureBlobContainer
```

Comment:
Please double confirm whether we should use `blob` or `file`. I'm not sure which one(or maybe both) support storage static website. Later we need to expose uploaded files as static site for public.

Reply:
I'm not 100% sure which one to use. but `blob container` is a feasible solution according to the doc here: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website-how-to?tabs=azure-portal